.github/workflows/pr.yaml

```diff
@@ -0,0 +1,130 @@
+on: pull_request
+name: Review
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    name: Changelog should be updated
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Git fetch
+        run: git fetch
```

Comment:
Shoule this fetch have been `git fetch origin/${{ github.base_ref }}` so only the branch you compare with belowe is fetched?

Reply:
Makes sense to simply fetch `base_ref`. 

Adjusted to use 

`git fetch origin ${{ github.base_ref }}:${{ github.base_ref }}`